presto-native-execution/presto_cpp/main/connectors/PrestoToVeloxConnector.cpp

```diff
@@ -1493,6 +1493,114 @@ IcebergPrestoToVeloxConnector::createConnectorProtocol() const {
   return std::make_unique<protocol::iceberg::IcebergConnectorProtocol>();
 }
 
+#ifdef PRESTO_ENABLE_ICEBERG_NATIVE_INSERTION
+
+std::unique_ptr<velox::connector::ConnectorInsertTableHandle>
```

Comment:
Is it possible to move IcebergPrestoToVeloxConnector to its own file ?
We could do that to the remaining PrestoToVeloxConnectors as well as a follow-up cleanup.